### ISSUE TITLE
SF PS module seems to require hard-coded FIPS feature-awarenes

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#62]
* 10.3.0.157
* 1.5.1.1    SolidFire.Core 

```
{
	"method": "GetFipsReport",
    "params": {},
    "id": 1
}
```
Response:

```
{
    "id": 1,
    "result": {
        "errorNodes": [],
        "nodes": [
            {
                "httpsEnabled": true,
                "nodeID": 17
            },
            {
                "httpsEnabled": true,
                "nodeID": 28
            },
            {
                "httpsEnabled": true,
                "nodeID": 34
            },
            {
                "httpsEnabled": true,
                "nodeID": 23
            }
        ]
    }
}
```
But `Get-SFFeatureStatus` doesn't have FIPS:

```
PS /home/sean> Get-SFFeatureStatus                                                                                                                                                                                                                                                                                                                                                      

Enabled Feature
------- -------
   True Vvols

```

**Comment by Salmanius** [comment]
You will be happy to know that this was actually caught when adding support for the SnapMirror Feature checking, and will be included in the 1.6 release.

**Comment by arjun960** [comment]
@scaleoutsean The issue is fixed and now get-sffeaturestatus will display every features(vvols, snapmirror, fips, fipsdrives) status. The changes will be available in the next SDK release.

Closing the issue.
